src/elf/parser.rs

```diff
@@ -416,6 +416,25 @@ impl ElfParser {
         find_sym(symtab, strtab, addr, st_type)
     }
 
+    /// Calculate the file offset of the given symbol.
+    ///
+    /// # Notes
+    /// It is the caller's responsibility to ensure that the symbol's section
+    /// index is not `SHN_UNDEF`.
+    fn file_offset(&self, shdrs: &[Elf64_Shdr], sym: &Elf64_Sym) -> Result<u64> {
+        debug_assert_ne!(sym.st_shndx, SHN_UNDEF);
```

Comment:
I tried to google what happens when ELF has more than 64K ELF sections and we have ELF symbol for one of those high-numbered ELF sections. I failed after few simple tries, but it will probably byte us at some point with huge production binaries. So let's at least file an issue to investigate later?

Reply:
Yes, good point. I put that under the umbrella of #64 (https://github.com/libbpf/blazesym/issues/64#issuecomment-1789837528) as well.